02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>Is this related to slicing hierarchies?</p><p>I suspect it might be in the wrong spot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>This belongs here, but how this relates to slicing hierarchies is getting a bit lost in the weeds of <code>create_parameter_hierarchy</code> and <code>at</code>  and such.</p><p> </p><p>It would be worth seeing how to make this and the above more streamlined (the <code>filter</code>  and <code>isin</code>  is a bit advanced if the person hasn't learned about measures yet...)   </p>

Reply:
<p>I removed this section as introducing <code>create_parameter_hierarchy_from_column</code> seems a bit too complex at this point in time of the hierarchies series. We may re-introduce this content in the next part 3 hierarchies notebook that will refer to source simulations and parameter hierarchies (is what I'm thinking as of now, but this may change) </p>